PWGLF/DataModel/LFResonanceTables.h

```diff
@@ -0,0 +1,149 @@
+DECLARE_SOA_COLUMN(Indices, indices, int[2]);                          //! Field for the track indices to remove auto-correlations
+DECLARE_SOA_COLUMN(Sign, sign, int8_t);                                //! Sign of the track charge
+DECLARE_SOA_COLUMN(TPCNClsCrossedRows, tpcNClsCrossedRows, uint8_t);   //! Number of TPC crossed rows
+DECLARE_SOA_COLUMN(DcaXY, dcaXY, float);                               //! DCA_xy
+DECLARE_SOA_COLUMN(DcaZ, dcaZ, float);                                 //! DCA_z
+DECLARE_SOA_COLUMN(X, x, float);                                       //! x position of the track
+DECLARE_SOA_COLUMN(Alpha, alpha, float);                               //! alpha position of the track
+DECLARE_SOA_COLUMN(TPCPIDselectionFlag, tpcPIDselectionFlag, uint8_t); //! TPC PID selection
+DECLARE_SOA_COLUMN(TOFPIDselectionFlag, tofPIDselectionFlag, uint8_t); //! TOF PID selection
+DECLARE_SOA_COLUMN(TPCnSigmaPi, tpcNSigmaPi, float);                   //! Pion TPC nSigma
+DECLARE_SOA_COLUMN(TPCnSigmaKa, tpcNSigmaKa, float);                   //! Kaon TPC nSigma
+DECLARE_SOA_COLUMN(TPCnSigmaPr, tpcNSigmaPr, float);                   //! Proton TPC nSigma
+DECLARE_SOA_COLUMN(TOFnSigmaPi, tofNSigmaPi, float);                   //! Pion TOF nSigma
+DECLARE_SOA_COLUMN(TOFnSigmaKa, tofNSigmaKa, float);                   //! Kaon TOF nSigma
+DECLARE_SOA_COLUMN(TOFnSigmaPr, tofNSigmaPr, float);                   //! Proton TOF nSigma
+DECLARE_SOA_COLUMN(DaughDCA, daughDCA, float);                         //! DCA between daughters
+DECLARE_SOA_COLUMN(MLambda, mLambda, float);                           //! The invariant mass of V0 candidate, assuming lambda
+DECLARE_SOA_COLUMN(MAntiLambda, mAntiLambda, float);                   //! The invariant mass of V0 candidate, assuming antilambda
+DECLARE_SOA_COLUMN(TransRadius, transRadius, float);                   //! Transverse radius of the decay vertex
+DECLARE_SOA_COLUMN(DecayVtxX, decayVtxX, float);                       //! X position of the decay vertex
+DECLARE_SOA_COLUMN(DecayVtxY, decayVtxY, float);                       //! Y position of the decay vertex
+DECLARE_SOA_COLUMN(DecayVtxZ, decayVtxZ, float);                       //! Z position of the decay vertex
+} // namespace resodaughter
+DECLARE_SOA_TABLE(ResoDaughters, "AOD", "RESODAUGHTERS",
+                  o2::soa::Index<>,
+                  resodaughter::ResoCollisionId,
+                  resodaughter::Pt,
+                  resodaughter::Px,
+                  resodaughter::Py,
+                  resodaughter::Pz,
+                  resodaughter::Eta,
+                  resodaughter::Phi,
+                  resodaughter::PartType,
+                  resodaughter::TempFitVar,
+                  resodaughter::Indices,
+                  resodaughter::Sign,
+                  resodaughter::TPCNClsCrossedRows,
+                  resodaughter::DcaXY,
+                  resodaughter::DcaZ,
+                  resodaughter::X,
```

Comment:
Here you could use `aod::X`

Reply:
That's new for me. Thanks for letting me know!